examples/swapi/src/PersonById.re

```diff
@@ -0,0 +1,48 @@
+let ste = ReasonReact.string;
+
+type action =
+  | ChangeId(string);
+
+type state = {id: string};
+
+let handleClick = event =>
```

Comment:
Is this used? 🤔

Reply:
Let me just say, this is awesome. First time doing this so, really cool to get quick helpful feedback. Thanks. 

@emmenko will review your comment in a sec. Thanks.

Reply:
removed unused handle click